central/notifiers/microsoftsentinel/sentinel.go

```diff
@@ -79,15 +102,36 @@ func (s sentinel) ProtoNotifier() *storage.Notifier {
 }
 
 func (s sentinel) Test(ctx context.Context) *notifiers.NotifierError {
-	alert := s.getTestAlert()
+	if s.sentinel().GetAuditLogDcrConfig().GetEnabled() {
+		err := s.SendAuditMessage(ctx, s.getTestAuditLogMessage())
+		if err != nil {
+			return notifiers.NewNotifierError("could not send audit message to sentinel", err)
+		}
+	} else {
+		log.Info("audit message are disabled, test audit message was not send to sentinel")
+	}
 
-	err := s.AlertNotify(ctx, alert)
-	if err != nil {
-		return notifiers.NewNotifierError("could not send event", err)
+	if s.sentinel().GetAlertDcrConfig().GetEnabled() {
```

Comment:
I think we need to consolidate how we get these "Enabled" checks.

You have `AuditLoggingEnabled()`
there are also checks of the form `s.notifier.GetMicrosoftSentinel().GetAuditLogDcrConfig().GetEnabled()`
then you have checks of the form `s.sentinel().GetAuditLogDcrConfig().GetEnabled()`

The best I can tell is they all are evaluating the same thing on the same notifier.  So maybe simpler to have a function like `AuditLoggingEnabled()` and create a similar one for alerts.

Reply:
`sentinel()` is a getter which executes `s.notifier.GetMicosoftSentinel`. Removed it because it caused confusion.